libs/remix-ui/run-tab/src/lib/components/recorderCardUI.tsx

```diff
@@ -22,24 +25,45 @@ export function RecorderUI (props: RecorderProps) {
         <div className="d-flex">
           <label className="mt-1 udapp_recorderSectionLabel">Transactions recorded</label>
           <div className="ml-2 mb-2 badge badge-pill badge-primary" title="The number of recorded transactions">{props.count}</div>
+        <OverlayTrigger placement={'right'} overlay={
+          <Tooltip className="text-nowrap" id="info-recorder">
+            <span>Save transactions (deployed contracts and function executions) and replay then in an
```

Comment:
`replay then in an another environment.` => `replay them in another environment.`